scenarios/pravega-flink-connector-sql-samples/src/main/java/io/pravega/connectors/nytaxi/MaxTravellersPerDestination.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.connectors.nytaxi;
+
+import io.pravega.connectors.flink.FlinkPravegaJsonTableSource;
+import io.pravega.connectors.flink.PravegaConfig;
+import io.pravega.connectors.nytaxi.common.TripRecord;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.streaming.api.TimeCharacteristic;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.api.java.StreamTableEnvironment;
+import org.apache.flink.table.api.java.Tumble;
+import org.apache.flink.table.sources.tsextractors.ExistingField;
+import org.apache.flink.table.sources.wmstrategies.BoundedOutOfOrderTimestamps;
+import org.apache.flink.types.Row;
+
+import java.net.URI;
+
+import static io.pravega.connectors.nytaxi.common.Constants.*;
+
+/**
+ * Find maximum number of travellers who travelled to a destination point for a given window interval
+ */
+
+@Slf4j
+public class MaxTravellersPerDestination {
```

Comment:
Added abstract class and renamed the class `PopularDestination` to `PopularDestinationQuery` since it is the only class that is using SQL whereas the other classes are using Table API